export const handleError = (err) => {
    return err.message || "An error ocurred"
}